Description: Force load bash completion plugins
 New style of Debian bash completion use load on demand of
 files. Unfortunately, cme plugins come with bash completions
 snippets that are not loaded with this mechanism. Hence, these lines
 circumvent delayed for the bash completions of cme plugins
 .
 This patch is Debian specific.. Well I think so...
Forwarded: not-needed
Author: dod
--- a/contrib/bash_completion.cme
+++ b/contrib/bash_completion.cme
@@ -12,6 +12,15 @@
 
 global_options='--force-load --create --trace --quiet --file --verbose --canonical'
 
+# New style of Debian bash completion use load on demand of
+# files. Unfortunately, cme plugins come with bash completions
+# snippets that are not loaded with this mechanism. Hence, these lines
+# circumvent delayed for the bash completions of cme plugins
+for file in /usr/share/bash-completion/completions/cme[-_]*
+do
+    . $file
+done
+
 _cme_models()
 {
    MODELS=$(perl -MConfig::Model::Lister -e'print Config::Model::Lister::models;')
